src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2254,11 +2189,108 @@ private BoundExpression BindNonMethod(SimpleNameSyntax node, Symbol symbol, Bind
                     }
 
                 case SymbolKind.RangeVariable:
+                    // The type calculation here should be kept in sync with logic in BindLeftIdentifierOfPotentialColorColorMemberAccess.
```

Comment:
Comment refers to `case SymbolKind.Field:` a few lines above (line 2171)
~~Should a similar comment be left in the field case?~~
~~`BindLeftIdentifierOfPotentialColorColorMemberAccess` has a case for fields when computing type~~

Never mind. I see it's in `BindFieldAccess` #Closed